src/Users.js

```diff
@@ -253,13 +254,23 @@ class Users extends React.Component {
     }
   }
 
+  fetchLoans = props => {
```

Comment:
I think the method is about to generate overdue loans report where fetching of loans is a kind of implementation detail. Please come up with another name.

Reply:
Thanks for the comment. What about 'loansToReport'?

Reply:
@SergiyVSergiyenko usually the function should contains an action name - call, find, get, etc. So it looks like 'loansToReport' can be improved